patches/expose-flags.patch

```diff
@@ -36,6 +36,11 @@ index 96370d4ac35a9..6429e45f5013e 100644
 +     "Shows punycode for IDN domains to mitigate IDN homograph attacks. "
 +     "Defaults to disabled. This feature is provided by hardened-chromium.",
 +     kOsAll, FEATURE_VALUE_TYPE(url::kShowPunycodeDomains)},
++    {"enable-gssapi", "Enable GSSAPI Authentication",
++     "Enables GSSAPI for authentication. WARNING! This can cause the "
++     "network service sandbox to become persistently disabled, enable only "
++     "if absolutely necessary. This flag is provided by hardened-chromium.",
++     kOsLinux, SINGLE_DISABLE_VALUE_TYPE("enable-gssapi")},
```

Comment:
from the docs the SINGLE_DISABLE type is 

    // A default enabled feature with a single flag value to disable it.

shouldn't this be default disabled?